### DISCUSSION TITLE
UNO Platform - Build a Todo App - Part 3

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#6] [General]
# UNO Platform - Build a Todo App - Part 3

In the third part of our small mini series: "Building an todo app with the Uno Platform" we will dive deeper into creating a modal dialog where we can enter the details of our todo item. We will see how we can import additional Nuget packages and how we can leverage validation.

https://steven-giesel.com/blogPost/a3179d55-d5be-48ba-b570-ee7d494a8b21

**Comment by zy8848** [comment]
Nice blog! Very useful resource for learn Uno platform

**Reply by linkdotnet** [reply]
Thank you very much for your support!
